go.mod

```diff
@@ -36,7 +36,7 @@ require (
 	k8s.io/utils v0.0.0-20240310230437-4693a0247e57
 	open-cluster-management.io/addon-framework v0.9.1-0.20240419070222-e703fc5a2556
 	open-cluster-management.io/api v0.13.1-0.20240521030453-9d94703b9eba
-	open-cluster-management.io/sdk-go v0.13.1-0.20240520073308-f18d198a844d
+	open-cluster-management.io/sdk-go v0.13.1-0.20240605032202-2a0633ddf34e
```

Comment:
@skeeey please update to latest commit: https://github.com/open-cluster-management-io/sdk-go/commit/874c2c7ac523ce73503b8322611b17510363c33f

there should no code change for your PR, just upgrade the dependency to get this fix: https://github.com/open-cluster-management-io/sdk-go/pull/52

Reply:
done